Makefile

```diff
@@ -0,0 +1,13 @@
+setup:
+	cp .env.example .env
```

Comment:
Should not be executed when there is an .env file

Reply:
I've just copied what was in the README to this Makefile. Do you have a specific suggestion?

Reply:
Fine for now. Will update it later to something like this.
https://github.com/minvws/nl-uzipoc-php-laravel-demo/blob/main/Makefile